### ISSUE TITLE
[APP SUBMITTED]: ThreadPoolExecutor-0_1 :: [fa36303] Missing time zone for network: DC Universe. Check valid network is set in indexer (theTVDB) before filing issue.

### ISSUE THREAD

**Issue by roxasrini** [SickChill/SickChill#5089]
### INFO
Python Version: **2.7.14 (default, May 28 2018, 15:25:50) [GCC 4.8.3 20140106 (prerelease)]**
Operating System: **Linux-3.10.102-armv7l-with-glibc2.4**
Locale: None
Branch: **master**
Commit: SickChill/SickChill@fa36303a1f48e9862729b0f1a74f83de81bc2522
No Log available with ERRORS:
### ERROR
```
ThreadPoolExecutor-0_1 :: [fa36303] Missing time zone for network: DC Universe. Check valid network is set in indexer (theTVDB) before filing issue.
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by Tiwas** [comment]
### INFO
Python Version: **2.7.14 (v2.7.14:84471935ed, Sep 16 2017, 20:25:58) [MSC v.1500 64 bit (AMD64)]**
Operating System: **Windows-10-10.0.17763**
Locale: cp1252
Branch: **master**
Commit: SickChill/SickChill@fa36303a1f48e9862729b0f1a74f83de81bc2522
No Log available with ERRORS:
### ERROR
```
DAILYSEARCHER :: [fa36303] Missing time zone for network: DC Universe. Check valid network is set in indexer (theTVDB) before filing issue.
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by dwightturner36** [comment]
### INFO
Python Version: **2.7.9 (default, Sep  9 2015, 14:50:03) [GCC 4.2.1 20070831 patched [FreeBSD]]**
Operating System: **FreeBSD-11.1-STABLE-amd64-64bit-ELF**
Locale: UTF-8
Branch: **master**
Commit: SickChill/SickChill@fa36303a1f48e9862729b0f1a74f83de81bc2522
No Log available with ERRORS:
### ERROR
```
ThreadPoolExecutor-0_3 :: [fa36303] Missing time zone for network: DC Universe. Check valid network is set in indexer (theTVDB) before filing issue.
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by meyersbs** [comment]
This appears to be a duplicate of #5084. I'm also getting this error.

**Comment by DrDrewZA** [comment]
Yup - anyone who added Titans to their show list is, I imagine...

**Comment by dokho1984** [comment]
Hello. I added Titans and I have this error ! Is this related to this specific show ?

**Comment by DrDrewZA** [comment]
Yup t is - but not because anything is wrong with the show.  The TV network - DCUniverse - is new and it's first show happens to be Titans.  As it's new it hasn't been added properly to either the TVDB or SickChill.  This is not a serious error and won't affect pretty much anything for you or anyone else (including me) that has this error.  Quite frankly, just ignore it while the devs get around to it - they have a few bigger fish to fry right now before they get to this one.

**Comment by dokho1984** [comment]
Ok. Thanks for your feedback. Have a good day.

**Comment by mmhere** [comment]
Now populated upstream as:  https://www.thetvdb.com/series/Titans-2018

That listing on thetvdb.com uses the string "DC Universe" (the **Network** field on that site) which will need to be added to http://sickchill.github.io/sb_network_timezones/network_timezones.txt

The latter URL is pulled as pairs {network, timezone} by sickbeard/networktimezones.py then pushed into the live cache.db and sourced throughout the running server from that last location.

Not sure what the actual timezone should be though...

The complaint about the missing timezone occurs only when starting (or restarting) SickChill.  Other than the red "Error" mark that appears in the web GUI things continue to run fine as mentioned in other comments above.

Cheers!
